packages/synapse-react-client/src/lib/style/base/_core.scss

```diff
@@ -1233,6 +1233,7 @@ hr ~ .SRC-wrapper {
 // Links created with MUI get their styling from the MUI theme.
 main,
 .modal,
+.MuiDialog-root,
```

Comment:
Apply link styling to non-MUI links within components, e.g. links in Id and Name columns within SynapseTable